### PR TITLE
Issue a severe error without AirflowNetwork model inputs when RoomAirModelType is AirflowNetwork

### PR DESCRIPTION
**Pull request by lgu1234**
Pull request overview
---------------------
Address #6086 
Issue a severe error for no inputs of the AirflowNetwork model when 	Room-Air Modeling Type = AirflowNetwork in the RoomAirModelType object.

### Work Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Title of PR should be user-synopsis style (clearly understandable in a standalone changelog context)
 - [x] At least one of the following appropriate labels must be added to this PR to be consumed into the changelog:
   - Defect: This pull request repairs a github defect issue.  The github issue should be referenced in the PR description
   - Refactoring: This pull request includes code changes that don't change the functionality of the program, just perform refactoring
   - NewFeature: This pull request includes code to add a new feature to EnergyPlus
   - Performance: This pull request includes code changes that are directed at improving the runtime performance of EnergyPlus
   - DoNoPublish: This pull request includes changes that shouldn't be included in the changelog

### Review Checklist
This will not be exhaustively relevant to every PR.
 - [x] Code style (parentheses padding, variable names)
 - [x] Functional code review (it has to work!)
 - [x] If defect, results of running current develop vs this branch should exhibit the fix
 - [x] CI status: all green or justified
 - [x] Performance: CI Linux results include performance check -- verify this
 - [x] Unit Test(s)
 - C++ checks:
   - [x] Argument types
   - [x] If any virtual classes, ensure virtual destructor included, other things
 - IDD changes n/a
 - [x] Documentation changes in place
 - [x] Changed docs build successfully

### CODE REVIEW

File: src/EnergyPlus/HeatBalanceAirManager.cc [hunk 1]
```diff
@@ -2936,12 +2936,11 @@ namespace HeatBalanceAirManager {
 				} else if ( SELECT_CASE_var == "AIRFLOWNETWORK" ) {
 					AirModel( ZoneNum ).AirModelType = RoomAirModel_AirflowNetwork;
 					AirModel( ZoneNum ).SimAirModel = true;
-					ValidateComponent( "RoomAirSettings:AirflowNetwork", cAlphaArgs( 2 ), IsNotOK, "GetRoomAirModelParameters" );
-					if ( IsNotOK ) {
-						ShowContinueError( "In " + cCurrentModuleObject + '=' + cAlphaArgs( 1 ) + '.' );
+					if ( GetNumObjectsFound( "AirflowNetwork:SimulationControl" ) == 0 ) {
+						ShowSevereError( "In " + cCurrentModuleObject + " = " + cAlphaArgs( 1 ) + ": " + cAlphaFieldNames( 3 ) + " = AIRFLOWNETWORK."  );
+						ShowContinueError( "The AirflowNetwork model inputs are required." );
```
Comment (mjwitte) [inline src/EnergyPlus/HeatBalanceAirManager.cc]: This checks for the presence of AirflowNetwork:SimulationControl.  Is that sufficient?  Looking at the I/O Ref introduction to "RoomAirflowNetwork Model" mentions that you need AirflowNetwork:Intrazone:Node and AirflowNetwork:Intrazone:Linkage.  I would add another sentence:

The model requires a complete set of other AirflowNetwork:* objects to form a complete network."  ?????

I would suggest changing the last line of the error message:
"AirflowNetwork:SimulationControl and related inputs are required."

**Review by mjwitte — COMMENTED** [review]
Tested with defect file.  Error works as expected.

File: src/EnergyPlus/HeatBalanceAirManager.cc (hunk 1 above)
Comment (lgu1234) [reply src/EnergyPlus/HeatBalanceAirManager.cc]: @mjwitte I like your suggestion. The RoomAirflowNetwork model requires a whole set of AirflowNetwork inputs. I am going to make changes.

**Comment by mjwitte** [comment]
@lgu1234 I made some minor changes to the error message and added a sentence to the I/O Ref.  Let me know if you agree with the changes.

**Comment by lgu1234** [comment]
@mjwitte I like the changes. Thanks.

**Comment by mjwitte** [comment]
@Myoldmopar Docs build locally for me, but they didn't show up [here](https://nrel.github.io/EnergyPlusBuildResults/EnergyPlus-9852ebc61ff1fe34f76347234ffacd322e79f3ae-x86_64-Linux-Ubuntu-14.04-gcc-4.8.html).  Does that mean there's a problem with the docs?

**Comment by Myoldmopar** [comment]
Maaaybe.  If they build locally for you, then don't let it hold you up from merging this in.  I'll make an issue to check it out.

**Comment by mjwitte** [comment]
No windows CI results, but this is very limited scope.  Merging.

**Pull request merged**